Description: move manpage to level 1 and set the option --date in pod2man command.
Author: Neil Williams <codehelp@debian.org>
Reviewed-by: Thiago Andrade Marques <thmarques@gmail.com>
Last-Update: 2019-12-22
Index: apt-rdepends/Makefile
===================================================================
--- apt-rdepends.orig/Makefile
+++ apt-rdepends/Makefile
@@ -30,20 +30,20 @@ mandir = ${prefix}/share/man
 INSTALL = /usr/bin/install -c
 INSTALL_DATA = ${INSTALL} -m 644
 
-all : apt-rdepends.8
+all : apt-rdepends.1
 
-apt-rdepends.8 : apt-rdepends
-	pod2man --section=8 --release=${VERSION} --center=" " $< > $@
+apt-rdepends.1 : apt-rdepends
+	pod2man --section=1 --release=${VERSION} --date="20 May 2013" --center=" " $< > $@
 
 install :
 	mkdir -p ${bindir}; \
 	${INSTALL} apt-rdepends ${bindir}; \
 
 install-man : all
-	mkdir -p ${mandir}/man8; \
-	${INSTALL_DATA} apt-rdepends.8 ${mandir}/man8;
+	mkdir -p ${mandir}/man1; \
+	${INSTALL_DATA} apt-rdepends.1 ${mandir}/man1;
 
 clean :
-	rm -f apt-rdepends.8
+	rm -f apt-rdepends.1
 
 .PHONY : all install clean
